jobs/_schema.js

```diff
@@ -23,7 +23,8 @@ var m1 = function () {
      * @return {void}
      */
     this.setup = function (context) {
-        // stub
+        context.state.TotalCount = 0;
```

Comment:
Can you please remove these from _schema.js and create another file for testing ? let keep schema test free. it should only reflect the API. we can change the code to ignore anything in this directory starting with "_" (currently it only ignores "_schema.js") to be able to add more test codes here.